spec/unit/shoulda/matchers/active_record/have_db_column_matcher_spec.rb

```diff
@@ -106,6 +106,24 @@
     end
   end
 
+  context 'with foreign_key option' do
+    it 'accepts a column that is has a foreign_key' do
+      expect(with_table(:author_id, :integer, foreign_key: true)).
+        to have_db_column(:author_id).with_options(foreign_key: true)
+    end
+
+    it 'accepts a column that is has a foreign_key with options' do
+      expect(with_table(:author_id, :integer, foreign_key: { to_table: :users })).
+        to have_db_column(:author_id).with_options(foreign_key: { to_table: :users })
+    end
+
+    it 'raises an error with the unknown options' do
```

Comment:
I'm not sure you need to repeat this test here; this is already taken care of on line 110.